apis/r/tests/testthat/test-SOMAExperiment-query-matrix-outgest.R

```diff
@@ -0,0 +1,209 @@
+test_that("matrix outgest with all results", {
+  pbmc_small <- get_data("pbmc_small", package = "SeuratObject")
```

Comment:
It's true that it's a rare occurance, but I've had it happen. It's more important with Bioc packages as there are times when CRAN rebuilds their Bioc collection at the time they're running a check for your package, which'll cause failures, but it's rare. I've just gotten in the habit of checking for suggested packages rather than assuming they're there

Reply:
You are supposed to do that.

CRAN was historically a little lax on checking it, but the language in _Writing R Extensions_ has gotten tighter.  It is still not fully enforced.  (IIRC it's an opt-in check when running `R CMD check`.)